### PR TITLE
enable eslint for installations

### PR DESCRIPTION
**Pull request by Feiyang1**

### CODE REVIEW

File: config/.eslintrc.json [hunk 1]
```diff
@@ -17,8 +17,9 @@
     "overrides": [
         {
             "files": [
-                "*test.ts",
-                "**/test/**/*.ts"
+                "**/*.test.ts",
+                "**/test/**/*.ts",
+                "**/testing/**/*.ts"
```
Comment (mmermerkaya) [inline config/.eslintrc.json]: I see code under `testing/` as regular code that is used for testing. I don't think we should disable these two rules under `testing/`.

File: packages/installations/src/helpers/idb-manager.ts [hunk 2]
```diff
@@ -33,6 +33,8 @@ const dbPromise: Promise<DB> = openDb(
     switch (upgradeDB.oldVersion) {
       case 0:
         upgradeDB.createObjectStore(OBJECT_STORE_NAME);
+        break;
```
Comment (mmermerkaya) [inline packages/installations/src/helpers/idb-manager.ts]: Lack of `break` statements here is intentional. Please see the comment above the `switch` statement.

Comment (Feiyang1) [reply packages/installations/src/helpers/idb-manager.ts]: oops. will fix.

File: config/.eslintrc.json (hunk 1 above)
Comment (Feiyang1) [reply config/.eslintrc.json]: Okay. will remove it.

**Pull request merged**